### PR TITLE
Data structure name refactored to screen configuration

### PR DESCRIPTION
**Pull request by javierFernandezMendez**

### CODE REVIEW

**Review by daniel-grana — APPROVED** [review]

File: src/app/main/data/lists/01.list-item-text/list-item-text.component.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class ListItemTextComponent {
   itemIcon: any;
 
   @ViewChild('sidenavComp', { static: false })
-  dataStructure: DataStructureComponent;
+  dataStructure: ScreenConfigurationComponent;
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/lists/01.list-item-text/list-item-text.component.ts:50]: I think this variable dataStructure is not necessary

File: src/app/main/data/lists/03.list-item-card/list-item-card.component.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ export class ListItemCardComponent {
   @ViewChild('actionText2', { static: false })
   actionText2: any;
   @ViewChild('sidenavComp', { static: false })
-  dataStructure: DataStructureComponent;
+  dataStructure: ScreenConfigurationComponent;
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/lists/03.list-item-card/list-item-card.component.ts:66]: I think this variable dataStructure is not necessary

File: src/app/main/data/lists/04.list-item-card-image/list-item-card-image.component.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ export class ListItemCardImageComponent {
   @ViewChild('rowHeight', { static: false })
   rowHeight: any;
   @ViewChild('sidenavComp', { static: false })
-  dataStructure: DataStructureComponent;
+  dataStructure: ScreenConfigurationComponent;
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/lists/04.list-item-card-image/list-item-card-image.component.ts:73]: I think this variable dataStructure is not necessary

File: src/app/main/data/table/02.fixed/table-fixed.component.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ export class TableFixedComponent {
   tableHeight: any;
 
   @ViewChild('sidenavComp', { static: false })
-  dataStructure: DataStructureComponent;
+  dataStructure: ScreenConfigurationComponent;
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/table/02.fixed/table-fixed.component.ts:19]: I think this variable dataStructure is not necessary

File: src/app/main/data/table/08.pagination/table-pagination.component.ts [hunk 5]
```diff
@@ -12,7 +12,7 @@ export class TablePaginationComponent {
   pageSizeOptionsInput: any;
 
   @ViewChild('sidenavComp', { static: false })
-  dataStructure: DataStructureComponent;
+  dataStructure: ScreenConfigurationComponent;
```
Comment (patricia-martinez-imatia) [inline src/app/main/data/table/08.pagination/table-pagination.component.ts:15]: I think this variable dataStructure is not necessary

**Review by patricia-martinez-imatia — CHANGES_REQUESTED** [review]

**Review by patricia-martinez-imatia — APPROVED** [review]

**Review by daniel-grana — APPROVED** [review]

**Pull request merged**